prowler/providers/aws/services/awslambda/awslambda_function_vpc_is_in_multiple_az/awslambda_function_vpc_is_in_multiple_az.metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+  "Provider": "aws",
+  "CheckID": "awslambda_function_vpc_is_in_multiple_az",
```

Comment:
```suggestion
  "CheckID": "awslambda_function_vpc_multi_az",
```

Reply:
The rest of the new checks related to multiple Availability Zones are called `*_multiple_az`, don't you think they should still be called that to keep the format?

Reply:
I think `multi_az` is more aligned with the cloud models.